### PR TITLE
Add dns release notes to plugins

### PR DESCRIPTION
**Pull request by jamesesilvia**
## Description
add dns plugin release notes
also fixup DNS App ID info note endind

## Testing
Screenshot 2023-12-15 at 2 19 42 PM
Screenshot 2023-12-15 at 2 19 36 PM

### CODE REVIEW

File: docs/plugin_dns_app_id.md [hunk 1]
```diff
@@ -341,6 +341,16 @@ exit
 The plugin must be updated to version 3.1.3 or later prior to [upgrading the conductor to SSR version 5.4.0.](intro_upgrade_considerations.md#plugin-config-generation-changes)
 :::
 
+### Release 3.3.0
+
+**Release Date:** Dec 15, 2023
+
+#### New Features and Improvements
+
+- **PLUGIN-1842** Pillar data scaling
```
Comment (agrawalkaushik) [inline docs/plugin_dns_app_id.md:350]: For consistency, we've used the following for other plugins with this fix

```
#### New Features and Improvements
- **PLUGIN-1839** Reduce time to apply salt states in large scale deployments

By using Saltstack data files, the time to apply high states across all assets is significantly reduced.

```

Comment (agrawalkaushik) [reply docs/plugin_dns_app_id.md:350]: I think its probably best to re-use that (you can use the new JIRA).

File: docs/plugin_dns_app_id.md [hunk 2]
```diff
@@ -341,6 +341,16 @@ exit
 The plugin must be updated to version 3.1.3 or later prior to [upgrading the conductor to SSR version 5.4.0.](intro_upgrade_considerations.md#plugin-config-generation-changes)
 :::
 
+### Release 3.3.0
+
+**Release Date:** Dec 15, 2023
```
Comment (MichaelBaj) [inline docs/plugin_dns_app_id.md:346]: Has this plugin already been released? If not, this date is incorrect.

File: docs/plugin_dns_cache.md [hunk 3]
```diff
@@ -217,6 +217,16 @@ Verify that the dns-cache network interface (default `dns-cache-intf`) is UP.
 The plugin must be updated to version 3.2.1 or later prior to [upgrading the conductor to SSR version 5.4.0.](intro_upgrade_considerations.md#plugin-config-generation-changes)
 :::
 
+### Release 3.5.0
+
+**Release Date:** Dec 15, 2023
```
Comment (MichaelBaj) [inline docs/plugin_dns_cache.md:222]: Same comments above apply here as well.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/plugin_dns_app_id.md (hunk 1 above)
Comment (jamesesilvia) [reply docs/plugin_dns_app_id.md]: took your suggestion and reworded a bit. please check

File: docs/plugin_dns_app_id.md (hunk 2 above)
Comment (jamesesilvia) [reply docs/plugin_dns_app_id.md]: updated to today

**Review by agrawalkaushik — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**